src/EA.Weee.RequestHandlers/Mappings/ReturnMap.cs

```diff
@@ -42,7 +42,8 @@ public ReturnData Map(ReturnQuarterWindow source)
                 SubmittedBy = source.Return.SubmittedBy?.FullName,
                 SubmittedDate = source.Return.SubmittedDate,
                 ReturnReportOns = source.ReturnReportOns.Select(r => new ReturnReportOn(r.ReportOnQuestionId, r.ReturnId)).ToList(),
-                ReturnStatus = mapper.Map<ReturnStatus>(source.Return.ReturnStatus)
+                ReturnStatus = mapper.Map<ReturnStatus>(source.Return.ReturnStatus),
```

Comment:
Add ReturnMap tests to check check for the setting of the nil return bool

Reply:
Added [Theory] test